api/models/Notification.js

```diff
@@ -178,9 +179,12 @@ module.exports = bookshelf.Model.extend({
   },
 
   sendCommentPush: function (version) {
-    var comment = this.comment()
-    var path = url.parse(Frontend.Route.post(comment.relations.post)).path
-    var alertText = PushNotification.textForComment(comment, version)
+    const comment = this.comment()
+    const post = comment.relations.post
+    const group = post.relations.groups.first()
```

Comment:
this is where we would ideally figure out the best group to send the person to, though not sure its setup to customize the notification text per user just yet, so that can wait.

Reply:
Yeah, Loren and I had a deep dive into this. We determined the listing of groups when the post is created is maintained in the DB, so first is as good as any. Of course, we might have users that are seeing the post because of their membership to another group. Right now, they'll be redirected to `/all` on the front-end but yes, we could check their memberships and match that to the groups on a post as a slightly more relevant option